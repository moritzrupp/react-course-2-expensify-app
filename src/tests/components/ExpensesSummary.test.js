import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary with 1 expense correctly', () => {
    const wrapper = shallow(<ExpensesSummary
        expenseCount={1}
        expensesTotal={1223}
    />);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with 2 expenses correctly', () => {
    const wrapper = shallow(<ExpensesSummary
        expenseCount={2}
        expensesTotal={1255}
    />);
    expect(wrapper).toMatchSnapshot();
});