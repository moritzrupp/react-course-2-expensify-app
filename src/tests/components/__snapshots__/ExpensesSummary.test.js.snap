// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with 1 expense correctly 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
     totalling 
    $12.23
  </p>
</div>
`;

exports[`should render ExpensesSummary with 2 expenses correctly 1`] = `
<div>
  <p>
    Viewing 
    2
     
    expenses
     totalling 
    $12.55
  </p>
</div>
`;
